### PR TITLE
 bug 1679159. Allow setting of pingTimeout

### PR DESCRIPTION
**Pull request by jcantrill**
This PR fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1679159

It backports https://github.com/elastic/elasticsearch-js/pull/679 to allow setting of the pingTimeout for high latency environments.  This is really a band aid that should be resolved by resolving the network or es memory issues causing the latency

### CODE REVIEW

File: kibana/install.sh [hunk 1]
```diff
@@ -6,10 +6,48 @@ set -o pipefail
...
+        fi
+        if ((10#${ver1[i]} < 10#${ver2[i]}))
+        then
+            return 2
+        fi
+    done
+    return 0
+}
+
+if compare_versions "$KIBANA_VER" "5.6.13" "=" ; then
```
Comment (richm) [inline kibana/install.sh]: The `"="` argument is unused?
The problem is only in 5.6.13?

Comment (jcantrill) [reply kibana/install.sh]: Updated.  its a 5.6.x issue.. that was fixed in 6.x.  My concern was a version difference in the library but i changed it to be 5.6.13+

**Comment by richm** [comment]
flake?
/retest

**Comment by richm** [comment]
@jcantrill tests are failing for real - kibana is crashing: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_origin-aggregated-logging/1539/pull-ci-openshift-origin-aggregated-logging-master-full-integ-aws/221/artifacts/full-integ-aws/scripts/check-EFK-running/artifacts/kibana-769555c77b-zwxrl.kibana.log
```
fs.js:642
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: EACCES: permission denied, open '/usr/share/kibana/optimize/.babelcache.json'
    at Error (native)
    at Object.fs.openSync (fs.js:642:18)
    at Object.fs.writeFileSync (fs.js:1356:33)
    at save (/usr/share/kibana/node_modules/babel-register/lib/cache.js:45:16)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)
    at Module.runMain (module.js:613:11)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:160:9)
    at bootstrap_node.js:507:3
```

**Pull request merged**